go/vt/vtadmin/api.go

```diff
@@ -384,6 +386,7 @@ func (api *API) Handler() http.Handler {
 	router.HandleFunc("/migration/{cluster_id}/{keyspace}/launch", httpAPI.Adapt(vtadminhttp.LaunchSchemaMigration)).Name("API.LaunchSchemaMigration").Methods("PUT", "OPTIONS")
 	router.HandleFunc("/migration/{cluster_id}/{keyspace}/retry", httpAPI.Adapt(vtadminhttp.RetrySchemaMigration)).Name("API.RetrySchemaMigration").Methods("PUT", "OPTIONS")
 	router.HandleFunc("/migrations/", httpAPI.Adapt(vtadminhttp.GetSchemaMigrations)).Name("API.GetSchemaMigrations")
+	router.HandleFunc("/movetables/{cluster_id}/complete", httpAPI.Adapt(vtadminhttp.MoveTablesComplete)).Name("API.MoveTablesComplete")
```

Comment:
Any reason we're doing movetables and not reshard? Looks like this PR branch does not appear to have the Reshard work from https://github.com/vitessio/vitess/pull/16903, so that may be why. Can we add it here so that we have some uniformity? We can also do it in a follow-up if needed.

Reply:
I think `complete` can be done for Reshard. I can do that in a follow-up PR.